### PR TITLE
Build search in Payslip and PayslipPayment - Debug search in plants

### PR DESCRIPTION
**Pull request by thib44**
- Classical search and advanced search

### CODE REVIEW

File: app/models/payslip_payment.rb [hunk 1]
```diff
@@ -52,8 +52,11 @@
 class PayslipPayment < OutgoingPayment
   acts_as_affairable :payee, dealt_at: :to_bank_at, debit: false, class_name: 'PayslipAffair'
 
+
   # This method permits to add journal entries corresponding to the payment
   # It depends on the preference which permit to activate the "automatic bookkeeping"
+
+
```
Comment (Aquaj) [inline app/models/payslip_payment.rb]: Could you remove those empty lines ?

File: app/views/backend/payslip_payments/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - max = payslip_payments_max_payment.to_f
+    - min = payslip_payments_min_amount.to_f
+    - label = PayslipPayment.human_attribute_name(:amount)
+    = render('backend/shared/kujaku/range_slider',
+        max: max,
+        min: min,
+        step: 10,
+        label: label,
+        name: :amount)
+
```
Comment (Aquaj) [inline app/views/backend/payslip_payments/index.html.haml]: Could you remove those empty lines ?

File: app/controllers/backend/payslips_controller.rb [hunk 3]
```diff
@@ -4,6 +4,8 @@ class PayslipsController < Backend::BaseController
 
     unroll :number, :amount, :currency, :created_at, employee: :full_name
 
+    
+
```
Comment (Aquaj) [inline app/controllers/backend/payslips_controller.rb]: Could you remove those empty lines ?

**Comment by thib44** [comment]
@Aquaj  Done

**Comment by Aquaj** [comment]
Could you pull master and resolve the conflicts ?

File: config/locales/eng/views/backend/payslips.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+fra:
+  backend:
+    payslips:
+      index:
+        rule: Rule
+        partialy_rule: Partially rule
+        not_rule: Not rule
```
Comment (burisu) [inline config/locales/eng/views/backend/payslips.yml]: Please use standard ways of Ekylibre I18n and English first.

File: config/locales/fra/views/backend/payslips.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+fra:
+  backend:
+    payslips:
+      index:
+        rule: Réglé
+        partialy_rule: Partiellement réglé
+        not_rule: Non réglé
```
Comment (burisu) [inline config/locales/fra/views/backend/payslips.yml]: Please use standard ways of Ekylibre I18n and English first.

File: app/helpers/backend/payslips_helper.rb [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      end
+
+      code << select_tag(name, options_for_select(list, value), :id => custom_id, 'data-show-value' => "##{configuration[:id]}_")
+
+      code << ' ' << content_tag(:span, :manual_period.tl(start: date_field_tag("#{name}_started_on".to_sym, params["#{name}_started_on"], size: 10), finish: date_field_tag("#{name}_stopped_on".to_sym, params["#{name}_stoped_on"], size: 10)).html_safe, id: "#{configuration[:id]}_interval")
+
+      code.html_safe
+    end
+  end
+end
```
Comment (burisu) [inline app/helpers/backend/payslips_helper.rb]: Factorize code with `app/helpers/backend/payslip_payments_helper.rb` please

**Pull request closed without merging**